**Learning Track 2: Biomarker discovery**

**Central Dogma of Molecular Biology in Cancer**

 

The central dogma of molecular biology is a theory coined by (**Crick**, 1970) in which the DNA is replicated, transcribed to RNA, and translated to protein. A double strand of DNA that contains our genetic information is duplicated and the given identical copies are transcribed to a single-stranded sequence of RNA which moves from the nucleus to the cytoplasm. The RNA sequence is translated by ribosome into amino acids which form protein. The ribosome reads three bases, known as codons, and translates them into one amino acid.

 

There are many known types of RNA involved in the process of protein translation, such as messenger RNAs (mRNA) that carry the DNA in the nucleus to the ribosome, ribosomal RNAs (rRNA) that form the ribosomes, and codon translators (tRNA) (**Taylor** _et al_., 2021). However, recent studies stated that the percentage of those RNAs is low compared to non-coding RNAs (ncRNAs). Non-coding RNAs are not junk molecules, as previously thought, but rather important molecules that require careful interpretation and understanding and act as regulators in cellular responses. Three major ncRNAs play a significant role in cancer biology: circular RNAs (circRNAs), long non-coding RNAs (lncRNAs), and microRNAs (miRNAs) (**Saw** _et al.,_ 2021).

 

To begin with, there is an altered expression of circRNA found in tumors (**Hansen** _et al_., 2013; **Memczak**_** **et al_., 2013). Another type of non-coding RNA is long non-coding RNAs (lncRNAs). Long non-coding RNAs (lncRNAs) could contribute to tumor formation as a result of the abnormal methylation of DNA or histone modifications. This could affect the expression of long lncRNAs, resulting in cancer development (**Mattick**_** **et al_., 2023). Finally, the third type of non-coding RNA relating to cancer is microRNAs. MicroRNAs contribute to cancer development, such as the discovery of deleted miR-15 and miR-16 loci in B-cell chronic lymphocytic leukemia (**Calin** _et al_., 2002).<!--[if supportFields]><span style='mso-ascii-font-family:
"Times New Roman";mso-ascii-theme-font:major-bidi;mso-hansi-font-family:"Times New Roman";
mso-hansi-theme-font:major-bidi;mso-bidi-font-family:"Times New Roman";
mso-bidi-theme-font:major-bidi;color:#0E101A'><span style='mso-element:field-begin'></span>
ADDIN ZOTERO_ITEM CSL_CITATION
{&quot;citationID&quot;:&quot;VEF9O131&quot;,&quot;properties&quot;:{&quot;formattedCitation&quot;:&quot;(La
Ferlita et al., 2021)&quot;,&quot;plainCitation&quot;:&quot;(La Ferlita et al.,
2021)&quot;,&quot;noteIndex&quot;:0},&quot;citationItems&quot;:[{&quot;id&quot;:18588,&quot;uris&quot;:[&quot;http://zotero.org/users/local/0mTottLY/items/3I4IAKJH&quot;],&quot;itemData&quot;:{&quot;id&quot;:18588,&quot;type&quot;:&quot;article-journal&quot;,&quot;abstract&quot;:&quot;RNA-Seq
is a well-established technology extensively used for transcriptome profiling,
allowing the analysis of coding and non-coding RNA molecules. However, this
technology produces a vast amount of data requiring sophisticated computational
approaches for their analysis than other traditional technologies such as
Real-Time PCR or microarrays, strongly discouraging non-expert users. For this
reason, dozens of pipelines have been deployed for the analysis of RNA-Seq
data. Although interesting, these present several limitations and their usage
require a technical background, which may be uncommon in small research
laboratories. Therefore, the application of these technologies in such contexts
is still limited and causes a clear bottleneck in knowledge
advancement.&quot;,&quot;container-title&quot;:&quot;BMC
Bioinformatics&quot;,&quot;DOI&quot;:&quot;10.1186/s12859-021-04211-7&quot;,&quot;ISSN&quot;:&quot;1471-2105&quot;,&quot;issue&quot;:&quot;1&quot;,&quot;journalAbbreviation&quot;:&quot;BMC
Bioinformatics&quot;,&quot;page&quot;:&quot;298&quot;,&quot;source&quot;:&quot;BioMed
Central&quot;,&quot;title&quot;:&quot;RNAdetector: a free user-friendly
stand-alone and cloud-based system for RNA-Seq data
analysis&quot;,&quot;title-short&quot;:&quot;RNAdetector&quot;,&quot;volume&quot;:&quot;22&quot;,&quot;author&quot;:[{&quot;family&quot;:&quot;La
Ferlita&quot;,&quot;given&quot;:&quot;Alessandro&quot;},{&quot;family&quot;:&quot;Alaimo&quot;,&quot;given&quot;:&quot;Salvatore&quot;},{&quot;family&quot;:&quot;Di
Bella&quot;,&quot;given&quot;:&quot;Sebastiano&quot;},{&quot;family&quot;:&quot;Martorana&quot;,&quot;given&quot;:&quot;Emanuele&quot;},{&quot;family&quot;:&quot;Laliotis&quot;,&quot;given&quot;:&quot;Georgios
I.&quot;},{&quot;family&quot;:&quot;Bertoni&quot;,&quot;given&quot;:&quot;Francesco&quot;},{&quot;family&quot;:&quot;Cascione&quot;,&quot;given&quot;:&quot;Luciano&quot;},{&quot;family&quot;:&quot;Tsichlis&quot;,&quot;given&quot;:&quot;Philip
N.&quot;},{&quot;family&quot;:&quot;Ferro&quot;,&quot;given&quot;:&quot;Alfredo&quot;},{&quot;family&quot;:&quot;Bosotti&quot;,&quot;given&quot;:&quot;Roberta&quot;},{&quot;family&quot;:&quot;Pulvirenti&quot;,&quot;given&quot;:&quot;Alfredo&quot;}],&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2021&quot;,6,3]]}}}],&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}
<span style='mso-element:field-separator'></span></span><![endif]--> A case study is conducted by **La Ferlita** _et al.,_ 2021  <!--[if supportFields]><span
style='mso-ascii-font-family:"Times New Roman";mso-ascii-theme-font:major-bidi;
mso-hansi-font-family:"Times New Roman";mso-hansi-theme-font:major-bidi;
mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:major-bidi;
color:#0E101A'><span style='mso-element:field-end'></span></span><![endif]-->as they used RNAdetector free software to sequence IncRNAs.

_** **_

In conclusion, human non-coding RNAs could be used as biomarkers for the diagnosis and prediction of cancer. Using non-coding RNAs as cancer biomarkers holds immense promise for the advancements of the health sector. Their unique regulatory roles in gene expression and their differential expression in cancerous tissues make them valuable tools for advancing cancer detection and treatment.

 

 

 

 

 

**References**

<!--[if supportFields]><span style='mso-element:field-begin'></span><span
style='mso-spacerun:yes'> </span>ADDIN ZOTERO_BIBL
{&quot;uncited&quot;:[[&quot;http://zotero.org/users/local/0mTottLY/items/RJNJTKKF&quot;],[&quot;http://zotero.org/users/local/0mTottLY/items/W2KJESDA&quot;],[&quot;http://zotero.org/users/local/0mTottLY/items/CBEDEWA4&quot;],[&quot;http://zotero.org/users/local/0mTottLY/items/AAUZWAWS&quot;],[&quot;http://zotero.org/users/local/0mTottLY/items/ES8BL56Q&quot;],[&quot;http://zotero.org/users/local/0mTottLY/items/JQ32R3U8&quot;],[&quot;http://zotero.org/users/local/0mTottLY/items/H7MZTQQL&quot;]],&quot;omitted&quot;:[],&quot;custom&quot;:[]}
CSL_BIBLIOGRAPHY <span style='mso-element:field-separator'></span><![endif]-->Calin, G. A., Dumitru, C. D., Shimizu, M., Bichi, R., Zupo, S., Noch, E., Aldler, H., Rattan, S., Keating, M., Rai, K., Rassenti, L., Kipps, T., Negrini, M., Bullrich, F., & Croce, C. M. (2002). Frequent deletions and down-regulation of micro- RNA genes miR15 and miR16 at 13q14 in chronic lymphocytic leukemia. _Proceedings of the National Academy of Sciences of the United States of America_, _99_(24), 15524–15529. https\://doi.org/10.1073/pnas.242606799

Crick, F. (1970). Central Dogma of Molecular Biology. _Nature_, _227_(5258), 561–563. https\://doi.org/10.1038/227561a0

_Frequent deletions and down-regulation of micro- RNA genes miR15 and miR16 at 13q14 in chronic lymphocytic leukemia—PubMed_. (n.d.). Retrieved September 3, 2024, from https\://pubmed.ncbi.nlm.nih.gov/12434020/

Hansen, T. B., Jensen, T. I., Clausen, B. H., Bramsen, J. B., Finsen, B., Damgaard, C. K., & Kjems, J. (2013). Natural RNA circles function as efficient microRNA sponges. _Nature_, _495_(7441), 384–388. https\://doi.org/10.1038/nature11993

La Ferlita, A., Alaimo, S., Di Bella, S., Martorana, E., Laliotis, G. I., Bertoni, F., Cascione, L., Tsichlis, P. N., Ferro, A., Bosotti, R., & Pulvirenti, A. (2021). RNAdetector: A free user-friendly stand-alone and cloud-based system for RNA-Seq data analysis. _BMC Bioinformatics_, _22_(1), 298. https\://doi.org/10.1186/s12859-021-04211-7

Mattick, J. S., Amaral, P. P., Carninci, P., Carpenter, S., Chang, H. Y., Chen, L.-L., Chen, R., Dean, C., Dinger, M. E., Fitzgerald, K. A., Gingeras, T. R., Guttman, M., Hirose, T., Huarte, M., Johnson, R., Kanduri, C., Kapranov, P., Lawrence, J. B., Lee, J. T., … Wu, M. (2023). Long non-coding RNAs: Definitions, functions, challenges and recommendations. _Nature Reviews Molecular Cell Biology_, _24_(6), 430–447. https\://doi.org/10.1038/s41580-022-00566-8

Memczak, S., Jens, M., Elefsinioti, A., Torti, F., Krueger, J., Rybak, A., Maier, L., Mackowiak, S. D., Gregersen, L. H., Munschauer, M., Loewer, A., Ziebold, U., Landthaler, M., Kocks, C., le Noble, F., & Rajewsky, N. (2013). Circular RNAs are a large class of animal RNAs with regulatory potency. _Nature_, _495_(7441), 333–338. https\://doi.org/10.1038/nature11928

Saw, P. E., Xu, X., Chen, J., & Song, E.-W. (2021). Non-coding RNAs: The new central dogma of cancer biology. _Science China Life Sciences_, _64_(1), 22–50. https\://doi.org/10.1007/s11427-020-1700-9

Taylor, M. R., Simon, E. J., Dickey, J., Hogan, K. A., & Campbell, N. A. (2021a). Campbell Biology: Concepts & Connections (10th ed.). Pearson.

 

<!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--> 

 
